packages/_private/angular-demo/tsconfig.app.json

```diff
@@ -0,0 +1,14 @@
+/* To learn more about this file see: https://angular.io/config/tsconfig. */
```

Comment:
Not valid JSON.

Reply:
Statement as above stands:

Microsoft "specialities" and automatically generated via Angular-CLI. Don´t know if I want to change it, could lead to errors when upgrading the project via `ng update` later on.